### ISSUE TITLE
 Elevated serum concentration of vitamin B12

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10228]
**Preferred term label:**

**Synonyms**
Elevated plasma vitamin B12 level
Elevated serum vitamin B12 level
Elevated serum cobalamin level

**Definition (free text, please give PubMed ID)**
PMID: 14636871 **Elevated levels of serum cobalamin** may be a sign of a serious, even life-threatening, disease. Hematologic disorders like chronic myelogeneous leukemia, promyelocytic leukemia, polycythemia vera and also the hypereosinophilic syndrome can result in elevated levels of cobalamin. Not surprisingly, a rise of the cobalamin concentration in serum is one of the diagnostic criteria for the latter two diseases.

PMID: 20301287 Although no specific laboratory abnormality alone is diagnostic of ALPS, the detection of the following facilitates the diagnosis:
(...)  **Increased levels** of the ALPS-specific biomarkers: soluble IL-10, IL-18, FasL, and **vitamin B12 in plasma/serum**

**Parent term (use hpo.jax.org/app)**
Abnormal vitamin B12 level HP:0040126

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Autoimmune lymphoproliferative syndrome (ALPS) ORPHA:3261

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000016
- label: Elevated circulating vitamin B12 concentration
- parents: HP:0040126
- pmid: PMID:14636871|PMID:20301287
- def: Concentration of vitamin B12 in the blood circulation above the upper limit of normal.
- comment: Elevated levels of serum cobalamin may be a sign of a serious, even life-threatening, disease. Hematologic disorders like chronic myelogeneous leukemia, promyelocytic leukemia, polycythemia vera and also the hypereosinophilic syndrome can result in elevated levels of cobalamin.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10228
- synonyms:  Elevated plasma vitamin B12 level(NONE)|Elevated serum vitamin B12 level(NONE)|Elevated serum cobalamin level(NONE)
